app/src/main/java/com/hedvig/app/feature/embark/passages/datepicker/DatePickerViewModel.kt

```diff
@@ -20,5 +20,5 @@ class DatePickerViewModel : ViewModel() {
         showDatePicker.value = selectedEpochMillis
     }
 
-    private fun epochMillisToLocalDate(epochMillis: Long) = epochMillis.epochMillisToLocalDate()
+    private fun epochMillisToLocalDate(epochMillis: Long) = epochMillis.epochMillisToLocalDateTime()
```

Comment:
This function seems to be lying now 🙈 also, is it even needed? 🤔 we can probably just call the extension at its call site, right?